src/app/components/PopularPlaylist/components/PlaylistCover/PlaylistCover.css

```diff
@@ -0,0 +1,36 @@
+.popular-playlist-container {
+  background-color: var(--trans-black);
+  border-radius: 10px;
+  width: 142px;
+  height: 178px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.popular-playlist-image {
+  border-radius: 10px;
+  overflow: hidden;
+  width: 127px;
+  height: 108px;
+  margin: 10px ;
+}
+
+.popular-playlist-brief {
+  margin: 0px 10px;
+}
+.popular-playlist-name {
+  color: #fff;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+}
+
+.popular-playlist-description {
+  font-size: 8px;
+  font-weight: 400;
+  line-height: 10px;
+  font-style: normal;
+  color: #aaa;
+}
```

Comment:
```suggestion
}

@media (min-width: 768px) {
  .popular-playlist-container {
    padding: 0 10px 20px 10px;
    flex: 1;
  }

  .popular-playlist-image {
    width: 100%;
    height: 100%;
  }

  .popular-playlist-image img {
    min-width: 142px;
    width: 100%;
    height: 100%;
    object-fit: cover;
  }
}
```

I would add extra code for handling side of PlaylistCover in desktop. Becuase it looks for me weird that there are so much space between covers. So I prefer to stretch them a little.
Check it on picturesL before and after. What do you think?
<img width="1277" alt="Screenshot 2023-11-12 at 20 31 10" src="https://github.com/LorenaCapraru/song-sieve/assets/91835307/2f8dea25-6b38-4cb5-80f3-d4330fdc3ee0">
<img width="1298" alt="Screenshot 2023-11-12 at 20 30 50" src="https://github.com/LorenaCapraru/song-sieve/assets/91835307/665a74c0-7950-4f7f-be4e-54aa34f90cdf">

Reply:
I love the rectangular container, but the photo cover looks too cropped. Also, the header doesn't have enough space when changing desktop screen  - 
![image](https://github.com/LorenaCapraru/song-sieve/assets/113237479/6d6dfed8-d466-4deb-891d-1b5285a537bd) - 
![image](https://github.com/LorenaCapraru/song-sieve/assets/113237479/9499cd5a-d50c-418e-bf02-151f52643a49)
. I suggest to leave image size as it is but to expand containers: 

![image](https://github.com/LorenaCapraru/song-sieve/assets/113237479/ed80f4ef-7f5a-49ee-80b5-e90ffd1f2906). What do you think?

Reply:
Ok, We can leave it like that😉